x/committee/keeper/querier_test.go

```diff
@@ -0,0 +1,247 @@
+package keeper_test
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/kava-labs/kava/app"
+	"github.com/kava-labs/kava/x/committee"
+	"github.com/kava-labs/kava/x/committee/keeper"
+	"github.com/kava-labs/kava/x/committee/types"
+)
+
+const (
+	custom = "custom"
+)
+
+var testTime time.Time = time.Date(1998, time.January, 1, 0, 0, 0, 0, time.UTC)
+
+func NewCommitteeGenesisState(cdc *codec.Codec, gs committee.GenesisState) app.GenesisState {
+	return app.GenesisState{committee.ModuleName: cdc.MustMarshalJSON(gs)}
+}
+
+type QuerierTestSuite struct {
+	suite.Suite
+
+	keeper keeper.Keeper
+	app    app.TestApp
+	ctx    sdk.Context
+	cdc    *codec.Codec
+
+	querier sdk.Querier
+
+	addresses   []sdk.AccAddress
+	testGenesis types.GenesisState
+	votes       map[uint64]([]types.Vote)
+}
+
+func (suite *QuerierTestSuite) SetupTest() {
+	suite.app = app.NewTestApp()
+	suite.keeper = suite.app.GetCommitteeKeeper()
+	suite.ctx = suite.app.NewContext(true, abci.Header{})
+	suite.cdc = suite.app.Codec()
+	suite.querier = keeper.NewQuerier(suite.keeper)
+
+	_, suite.addresses = app.GeneratePrivKeyAddressPairs(5)
+	suite.testGenesis = types.NewGenesisState(
+		3,
+		[]types.Committee{
+			{
+				ID:               1,
+				Description:      "This committee is for testing.",
+				Members:          suite.addresses[:3],
+				Permissions:      []types.Permission{types.GodPermission{}},
+				VoteThreshold:    d("0.667"),
+				ProposalDuration: time.Hour * 24 * 7,
+			},
+			{
+				ID:               2,
+				Members:          suite.addresses[2:],
+				Permissions:      nil,
+				VoteThreshold:    d("0.667"),
+				ProposalDuration: time.Hour * 24 * 7,
+			},
+		},
+		[]types.Proposal{
+			{ID: 1, CommitteeID: 1, PubProposal: gov.NewTextProposal("A Title", "A description of this proposal."), Deadline: testTime.Add(7 * 24 * time.Hour)},
+			{ID: 2, CommitteeID: 1, PubProposal: gov.NewTextProposal("Another Title", "A description of this other proposal."), Deadline: testTime.Add(21 * 24 * time.Hour)},
+		},
+		[]types.Vote{
+			{ProposalID: 1, Voter: suite.addresses[0]},
+			{ProposalID: 1, Voter: suite.addresses[1]},
+			{ProposalID: 2, Voter: suite.addresses[2]},
+		},
+	)
+	suite.app.InitializeFromGenesisStates(
+		NewCommitteeGenesisState(suite.cdc, suite.testGenesis),
+	)
+
+	suite.votes = getProposalVoteMap(suite.keeper, suite.ctx)
+}
+
+func (suite *QuerierTestSuite) TestQueryCommittees() {
+	ctx := suite.ctx.WithIsCheckTx(false)
+	// Set up request query
+	query := abci.RequestQuery{
+		Path: strings.Join([]string{custom, types.QuerierRoute, types.QueryCommittees}, "/"),
+	}
+
+	// Execute query and check the []byte result
+	bz, err := suite.querier(ctx, []string{types.QueryCommittees}, query)
+	suite.NoError(err)
```

Comment:
we should be using `suite.Require().` everywhere because the default is an assertion.
```suggestion
	suite.Require().NoError(err)
```

Reply:
I'll punt this to an issue for now as most of our modules aren't using `Require()`.